```
        lint_test.go:86: 
            In file included from ../server/status/runtime_jemalloc.go:20:
            /go/native/x86_64-pc-linux-gnu/jemalloc/include/jemalloc/jemalloc.h:198:46: warning: unknown attribute 'alloc_size' ignored [-Wunknown-attributes]
            /go/native/x86_64-pc-linux-gnu/jemalloc/include/jemalloc/jemalloc.h:152:50: note: expanded from macro 'JEMALLOC_ALLOC_SIZE'
            /go/native/x86_64-pc-linux-gnu/jemalloc/include/jemalloc/jemalloc.h:201:46: warning: unknown attribute 'alloc_size' ignored [-Wunknown-attributes]
            /go/native/x86_64-pc-linux-gnu/jemalloc/include/jemalloc/jemalloc.h:153:56: note: expanded from macro 'JEMALLOC_ALLOC_SIZE2'
            /go/native/x86_64-pc-linux-gnu/jemalloc/include/jemalloc/jemalloc.h:207:5: warning: unknown attribute 'alloc_size' ignored [-Wunknown-attributes]
            /go/native/x86_64-pc-linux-gnu/jemalloc/include/jemalloc/jemalloc.h:152:50: note: expanded from macro 'JEMALLOC_ALLOC_SIZE'
            /go/native/x86_64-pc-linux-gnu/jemalloc/include/jemalloc/jemalloc.h:210:24: warning: unknown attribute 'alloc_size' ignored [-Wunknown-attributes]
            /go/native/x86_64-pc-linux-gnu/jemalloc/include/jemalloc/jemalloc.h:152:50: note: expanded from macro 'JEMALLOC_ALLOC_SIZE'
            /go/native/x86_64-pc-linux-gnu/jemalloc/include/jemalloc/jemalloc.h:216:27: warning: unknown attribute 'alloc_size' ignored [-Wunknown-attributes]
            /go/native/x86_64-pc-linux-gnu/jemalloc/include/jemalloc/jemalloc.h:152:50: note: expanded from macro 'JEMALLOC_ALLOC_SIZE'
            /go/native/x86_64-pc-linux-gnu/jemalloc/include/jemalloc/jemalloc.h:219:16: warning: unknown attribute 'alloc_size' ignored [-Wunknown-attributes]
            /go/native/x86_64-pc-linux-gnu/jemalloc/include/jemalloc/jemalloc.h:152:50: note: expanded from macro 'JEMALLOC_ALLOC_SIZE'
```


.... o_o"

--- a/build/builder.sh
+++ b/build/builder.sh
@@ -3,7 +3,7 @@
 set -euo pipefail
 
 image=cockroachdb/builder
-version=20201001-110640
+version=20201020-171515
 
 function init() {
   docker build --tag="${image}" "$(dirname "${0}")/builder"

--- a/build/builder/Dockerfile
+++ b/build/builder/Dockerfile
@@ -224,8 +224,11 @@ RUN curl -fsSL https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key ad
  && echo 'deb https://packages.cloud.google.com/apt cloud-sdk-xenial main' | tee /etc/apt/sources.list.d/gcloud.list \
  && curl -fsSL https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
  && echo "deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main" | tee /etc/apt/sources.list.d/google.list \
+ && echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list \
+ && curl https://bazel.build/bazel-release.pub.gpg | apt-key add - \
  && apt-get update
 
+# bazel - build system
 # ccache - speed up C and C++ compilation
 # lsof - roachprod monitor
 # netcat - roachprod monitor
@@ -237,6 +240,7 @@ RUN curl -fsSL https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key ad
 # chrome - ui
 # unzip - for installing awscli
 RUN apt-get install -y --no-install-recommends \
+    bazel-3.7.0 \
     ccache \
     google-cloud-sdk \
     lsof \

--- a/pkg/testutils/lint/lint_test.go
+++ b/pkg/testutils/lint/lint_test.go
@@ -1823,6 +1823,10 @@ func TestLint(t *testing.T) {
 			// Ignore types that can change by system.
 			stream.GrepNot(`pkg/util/sysutil/sysutil_unix.go:`),
 
+			// Ignore jemalloc issues warnings.
+			stream.GrepNot(`In file included from.*(start|runtime)_jemalloc\.go`),
+			stream.GrepNot(`include/jemalloc/jemalloc\.h`),
+
 			stream.GrepNot(`declaration of "?(pE|e)rr"? shadows`),
 			stream.GrepNot(`\.pb\.gw\.go:[0-9:]+: declaration of "?ctx"? shadows`),
 			stream.GrepNot(`\.[eo]g\.go:[0-9:]+: declaration of ".*" shadows`),

